src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+
+public class FigureSupplier {
+    ColorSupplier colorSupplier = new ColorSupplier();
+    Random random = new Random();
```

Comment:
The 'random' variable should be private and static if it's meant to be shared across instances of 'FigureSupplier'.